spec/spec_helper.rb

```diff
@@ -0,0 +1,2 @@
+# encoding: utf-8
```

Comment:
you can probably just move this include to your spec for one less file to manage.